### ISSUE TITLE
Allow/add additional technology from mods

### ISSUE THREAD

**Issue by frenzeldk** [milaq/sit#9]
There is this really awesome mod called [More Technologies](http://steamcommunity.com/sharedfiles/filedetails/?id=683851819). It would be crazy awesome, if the tech tree from this mod could be integrated.
If the tech tree data currently is pulled from the YML-files, this should be fairly easy, if not, it would require a rewrite (I'd be willing to participate!).
Oh and of course, this should be implemented as a toggle or fork.

**Comment by frenzeldk** [comment]
I just saw, you're parsing from the Wiki. I'd be willing to help rewrite the parser to use the YML-source file.

**Comment by milaq** [comment]
Hey mate,
thanks for your interest.

I am working on a way to extract technology information not only from the wiki but the gamefiles itself.
They are all available in `common/technology/`. Localisation is available in `technology/technology_l_english.yml` and `technology/triggers_effects_l_english.yml`

Localisation is in YAML which is easy to parse in python (which I am using for my JSON parser).
Technology defines themselves are in a weird Plaintext format which should be easily parsable too.

So as long as your mod uses the same structure (which I am assuming it does), I will extend SIT and sitparse to allow fetching data from other sources than the base game files.

For now, I would like to postpone this until I altered the parsing code to use the game files as source instead of the wiki.

I'll happily come back to you once this is done. But keep in mind that unfortunately I can't give you an ETA on this because in the coming weeks I have some important stuff to take care of in RL.

**Comment by milaq** [comment]
Requires #10

**Comment by Depact** [comment]
-- Comment ported to #11  --
